tests/interfaces/test_jax_qnode.py

```diff
@@ -73,11 +73,16 @@ def circuit(a):
             return qml.expval(qml.PauliZ(0))
 
         a = np.array(0.1, requires_grad=True)
-        circuit(a)
+
+        if interface == "jax-jit":
+            jax.jit(circuit)(a)
+        else:
+            circuit(a)

```

Comment:
Wait, were we not testing JIT for `interface="jax-jit"`?

Reply:
That's true just the interface but not jitted.